backend/src/services/release.ts

```diff
@@ -330,50 +339,29 @@ function getSemverQueryBounds(querySemver: string) {
   const semverRangeStandardised = semver.validRange(querySemver, { includePrerelease: false })
 
   if (!semverRangeStandardised) {
-    throw BadReq(`Semver range, '${querySemver}' is invalid `)
+    throw BadReq(`Semver range, '${querySemver}' is invalid.`)
   }
 
   const [expressionA, expressionB] = semverRangeStandardised.split(' ')
 
-  let lowerSemver: string = '',
-    upperSemver: string = '',
-    lowerInclusivity = false,
-    upperInclusivity = false
+  let lowerInclusivity, lowerSemver, upperInclusivity, upperSemver
```

Comment:
These are all `any`, they should be typed. If this was a JS project I wouldn't have a problem with defining multiple vars on a single line like this, but in TS we should either be providing a default value or explicitly typing it as `MyType | undefined`. As a result it's much more readable to define them on separate lines.
```
let var1
let var2
// etc
```

Reply:
Changed